### ISSUE TITLE
[api-extractor 7] generated *.api.json doesn't contains Class "constructor" info

### ISSUE THREAD

**Issue by adventure-yunfei** [Microsoft/web-build-tools#997]
Providing following .d.ts file:

```typescript
/** It's a class */
export class Class0 {
    /** It's a constructor */
    constructor(foo: Class0);
}
```

api-extractor@7.0.6 generates following .api.json file:

```json
{
  "metadata": {
    "toolPackage": "@microsoft/api-extractor",
    "toolVersion": "7.0.6",
    "schemaVersion": 1000
  },
  "kind": "Package",
  "name": "demo",
  "canonicalReference": "demo",
  "docComment": "",
  "members": [
    {
      "kind": "EntryPoint",
      "name": "",
      "canonicalReference": "",
      "members": [
        {
          "kind": "Class",
          "name": "Class0",
          "canonicalReference": "(Class0:class)",
          "docComment": "/**\n * It's a class\n */\n",
          "releaseTag": "Public",
          "members": [],
          "excerptTokens": [
            {
              "kind": "Content",
              "text": "export class "
            },
            {
              "kind": "Reference",
              "text": "Class0"
            },
            {
              "kind": "Content",
              "text": " "
            }
          ],
          "implementsTokenRanges": []
        }
      ]
    }
  ]
}
```

The "constructor" info is not included (while it is in api-extractor@6)

**Comment by pgonzal** [comment]
This is part of a list of node types that were procrastinated while we were ironing out the model:

Constructor
ConstructSignature
FunctionDeclaration
IndexSignature
TypeAliasDeclaration
VariableDeclaration

Should be pretty easy to add now.  I'll create a PR.

**Comment by pgonzal** [comment]
This will be fixed in PR https://github.com/Microsoft/web-build-tools/pull/998

**Comment by pgonzal** [comment]
@adventure-yunfei  This is fixed in the latest beta **api-extractor@7.0.7** and **api-documenter@7.0.9**
